setup.cfg

```
@@ -43,11 +43,12 @@ classifiers =
     Topic :: System :: Monitoring
     Framework :: Apache Airflow
 project_urls =
+    Bug Tracker=https://github.com/apache/airflow/issues
+    Changelog=https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html
```

```suggestion
    Release Notes=https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html
```

And maybe reorder it as well...